src/test/java/picard/vcf/GatherVcfsTest.java

```diff
@@ -47,33 +47,40 @@ public void setup() throws IOException {
     @DataProvider
     public Object[][] vcfshards() {
         return new Object[][]{
-                {Arrays.asList(shard1,shard2,shard3), vcf, 0},
-                {Arrays.asList(shard1,shard2_bad,shard3), vcf, 1},
-                {Arrays.asList(shard1,shard3,shard2), vcf, 1},
-                {Arrays.asList(shard3,shard1,shard2), vcf, 1} ,
-                {Arrays.asList(shard1_gz, shard2_gz, shard3), vcf_gz, 0},
-                {Arrays.asList(shard1_gz, shard2_bad_gz, shard3), vcf_gz, 1},
-                {Arrays.asList(shard1_gz, shard3_gz, shard2), vcf_gz, 1},
-                {Arrays.asList(shard3_gz, shard1_gz, shard2), vcf_gz, 1}
+                {Arrays.asList(shard1,shard2,shard3), vcf, 0, false},
```

Comment:
spacing here (and on the next 4 lines)
```suggestion
                {Arrays.asList(shard1, shard2, shard3), vcf, 0, false},
```